src/main/docs/guide/customSerializers.adoc

```diff
@@ -77,7 +77,7 @@ class MyJaxbElement3 {
 
 And set openapi properties to map classes to custom openapi schema classes:
 
-[configuartion]
```

Comment:
we want to use the `configuration` macro.

Reply:
fixed